packages/material-ui-lab/src/TabPanel/TabPanel.test.tsx

```diff
@@ -1,24 +1,29 @@
 import * as React from 'react';
 import { expect } from 'chai';
-import { getClasses, createMount, createClientRender, describeConformance } from 'test/utils';
+import { createMount, createClientRender, describeConformanceV5 } from 'test/utils';
 import TabPanel from './TabPanel';
+import classes from './tabPanelClasses';
 import TabContext from '../TabContext';
 
 describe('<TabPanel />', () => {
   const mount = createMount();
-  let classes: Record<string, string>;
   const render = createClientRender();
 
-  before(() => {
-    classes = getClasses(<TabPanel value="0" />);
-  });
-
-  describeConformance(<TabPanel value="0" />, () => ({
+  describeConformanceV5(<TabPanel value="0" />, () => ({
     classes,
     inheritComponent: 'div',
-    mount: (node) => mount(<TabContext value="0">{node}</TabContext>),
+    render,
+    mount: (node: any) => mount(<TabContext value="0">{node}</TabContext>),
     refInstanceof: window.HTMLDivElement,
-    skip: ['componentProp', 'reactTestRenderer'],
+    muiName: 'TabPanel',
+    skip: [
+      'componentProp',
+      'componentsProp',
+      'reactTestRenderer',
+      'themeDefaultProps',
+      'themeStyleOverrides',
```

Comment:
```suggestion
```

Let's enable this test, it will ensure that the `styleOverrides` from the theme work

Reply:
I enabled the test. The unit test passes on my computer. The karma test fails, however, due to the fact that a TabContext is not provided. I believe that's why I disabled the test. Do you have a suggestion how I can make the test pass without having to modify `describeConformanceV5`?

Reply:
I see, you can update the `render` param to be: `render: (node: any) => render(<TabContext value="0">{node}</TabContext>),` similar to the mount :) I've pushed the changes, it should be fine now :)

Reply:
Awesome, tnx :) I keep this in mind :)